jobs/JEVS_NFCENS_WAVE_GRID2OBS_PREP

```diff
@@ -56,8 +56,8 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 # Define COMIN/COMOUT variables     
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})} 
-export COMINgdas=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}
+export COMINnfcens=${COMINnfcens:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})} 
+export COMINobsproc=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}
```

Comment:
@SamiraArdani-NOAA Can you please change COMINobs to COMINobsproc in this line?

export COMINobsproc=${**COMINobsproc**:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}